Task13.cpp

```diff
@@ -0,0 +1,37 @@
+unsigned charToDigit(char ch)

```

Comment:
The function should be of type `int`, because if we do not enter into any of the for-loops, we should return `-1`, instead of `0`, because `0` is a valid number